src/main/java/co/refiere/resources/dataimport/DatabaseImporterResource.java

```diff
@@ -0,0 +1,154 @@
+package co.refiere.resources.dataimport;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Iterator;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+import org.glassfish.jersey.media.multipart.FormDataParam;
+
+import co.refiere.dao.PersonDao;
+import co.refiere.models.Person;
+
+@Path("/database")
+public class DatabaseImporterResource {
+
+  private static final String CSV_SPLIT_COMMA = ",";
+  private static final String CSV_EXTENSION = ".csv";
+  private static final String EXCEL_EXTENSION = ".xlsx";
+  private static final int NAME_POSITION_CSV = 0;
+  private static final int LAST_NAME_POSITION_CSV = 1;
+  private static final int EMAIL_POSITION_CSV = 2;
+  private static final int PHONE_NUMBER_POSITION_CSV = 3;
+  private static final int IDENTIFICATION_CARD_NUMBER_POSITION_CSV = 4;
+  private static final int IDENTIFICATION_CARD_NUMBER_ROW_NUMBER = 4;
+  private static final int PHONENUMBER_ROW_NUMBER = 3;
+  private static final int EMAIL_ROW_NUMBER = 2;
+  private static final int LASTNAME_ROW_NUMBER = 1;
+  private static final int NAME_ROW_NUMBER = 0;
+  private static final int FIRST_SHEET = 0;
+
+  @POST
+  @Path("/import")
+  @Consumes(MediaType.MULTIPART_FORM_DATA)
+  public void insertDataInDB2(@FormDataParam("file") InputStream uploadedInputStream,
```

Comment:
End-point not following Refiere endpoints standard, please move this logic to a proper service class, resource classes should not have business logic.